packages/wdio-config/tests/configparser.test.ts

```diff
@@ -728,9 +728,16 @@ describe('ConfigParser', () => {
             const configParser = await ConfigParserForTest()
             await configParser.addConfigFile(FIXTURES_CONF)
             configParser.merge({ spec: [INDEX_PATH, FIXTURES_CONF] })
-            configParser.merge({ exclude: [path.resolve(__dirname).replace(/\\/g, '/') + '/*'] })
-            const specs = configParser.getSpecs()
-            expect(specs).toHaveLength(2)
+
+            // first validate that the conf fixture 'spec' was merged successfully
+            expect(configParser.getSpecs()).toHaveLength(1)
+
+            configParser.merge({ exclude: [path.join(__dirname, '**', '*conf*')] })
+
+            // then after merging an exclude containing a glob pattern, validate that the exclude
+            // attribute contains multiple items and the filtering on the spec attribute works
+            expect(configParser.getConfig().exclude.length).toBeGreaterThan(0)
+            expect(configParser.getSpecs()).toHaveLength(0)
```

Comment:
this test was added via this PR: https://github.com/webdriverio/webdriverio/pull/4544/files but from my understanding, in its previous form, it was not actually validating what it stated... the test was passing even without the changes from https://github.com/webdriverio/webdriverio/pull/4544/files#diff-e0b205fb022f00db1b2dafbf906129bd7627eeea8db65d3be9285f6aa215c691R217-R221

The only reason it started failing in my PR is because i eliminated duplicated from `getSpecs()` so i tried to make it more robust